src/OrderImport/GLS/CartCarrierAssociation.php

```diff
@@ -40,7 +40,11 @@ public function __construct(AdapterInterface $glsAdapter)
 
     public function create(Cart $cart, $relayId)
     {
-        $relay_detail = $this->glsAdapter->getRelayDetail($relayId);
+        try {
+            $relay_detail = $this->glsAdapter->getRelayDetail($relayId);
+        } catch (Exception $e) {
```

Comment:
What is the goal of this catch?

Reply:
Firstly I throw Exception on nkmgls instead of return null with an error message to know that there is an issue with class path (or bas version). But the code is in the __contruct of Adapter and if the module nkmgls is not isntalled
Finally I decided to keep it in case of exception in $glsModule->getGlsProductCode()

Reply:
I mean this try-catch does nothing. 
If there is a try-catch, then the error is caught and is thrown again with the same message.
If there is no try-catch, then the error is not caught and it is raised above.